src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -64,6 +64,10 @@ public WorkflowMultiBranchProject(ItemGroup parent, String name) {
         return new WorkflowBranchProjectFactory();
     }
 
+    @Override protected String getSuccessfulDestination() {
+        return "computation/console";
```

Comment:
@jglick I did it [here](https://github.com/jenkinsci/cloudbees-folder-plugin/pull/58/files#diff-73ee53b9a4352457ad14440d3f6692c0R242) but it is strange that:

1. if you click on _Branch Indexing_ the url contains `computation/` but...
1. if you add a new Multibranch project, you are redirected to `indexing/` blah blah

IMO, the URLs in the left menu (actions) are not valid currently.